staging/src/k8s.io/kubectl/pkg/cmd/create/create_pdb.go

```diff
@@ -17,12 +17,22 @@ limitations under the License.
 package create
 
 import (
+	"context"
+	"fmt"
+	"regexp"
+
 	"github.com/spf13/cobra"
 
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
+	resourcecli "k8s.io/cli-runtime/pkg/resource"
```

Comment:
`resource` is good alias, please re-use aliases similar to other files, for consistency.